src/cmd/linuxkit/azure.go

```diff
@@ -307,9 +314,9 @@ func setVirtualMachineParameters(storageAccountName string, networkInterfaceID,
 				VMSize: compute.StandardDS1,
 			},
 			OsProfile: &compute.OSProfile{
-				ComputerName:  to.StringPtr("linuxkit"),
-				AdminUsername: to.StringPtr("dummyusername"),
-				AdminPassword: to.StringPtr("DummyPassword!123"),
+				ComputerName:  to.StringPtr(defaultComputerName),
+				AdminUsername: to.StringPtr(unusedAdminUsername),
+				AdminPassword: to.StringPtr(unusedPassword),
```

Comment:
Ah yes, ok. I think it is better to make the username and password "unused" and "unused" for now, if they can't be empty.